### ISSUE TITLE
Container breaks wireguard config

### ISSUE THREAD

**Issue by nils1323** [binhex/arch-sabnzbdvpn#27]
On start the container adds "," behind my address and allowedIps. 
My config before I start the container:
```
[Interface]
PostUp = '/root/wireguardup.sh'
PostDown = '/root/wireguarddown.sh'
PrivateKey = ***
Address = 172.26.88.11/32
DNS = ****

[Peer]
PublicKey = ***
AllowedIPs = 0.0.0.0/0
Endpoint = my.example.domain:9929
```
My config after I started the container:
```
[Interface]
PostUp = '/root/wireguardup.sh'
PostDown = '/root/wireguarddown.sh'
PrivateKey = ***
Address = 172.26.88.11/32,
DNS = ****

[Peer]
PublicKey = ***
AllowedIPs = 0.0.0.0/0,
Endpoint = my.example.domain:9929
```
each time I have to manually adjust my wireguard configs and remove those "," after starting the container otherwise the container wouldn't start because the wireguard config is invalid.
I'm running on unraid and I'm using OVPN.com as my VPN-Provider.

**Comment by binhex** [comment]
there is no semi colon ```;``` shown in your example above, either before or after the container has started?.

**Comment by nils1323** [comment]
I found the bug. I had multiple configs in my wireguard directory which were named vpn*.conf which lead to the first one beeing renamed to wg0.conf because of the install.sh skript line 131 selectiong the first .conf file. This clashed with your IPv6 removal script not handling whitespaces in the config very well (which I guess I fixed in my [PR](https://github.com/binhex/arch-int-vpn/pull/24)).
I fixed that by removing all other config files and suplying a config file without whitespaces
